server/py/services/api/crud/runtimes/nuclio/function.py

```diff
@@ -417,14 +418,22 @@ def _configure_serving_spec(
                     + "in the returned json, and check if the function runs successfully. "
```

Comment:
You should also change the check for maximal length that happens right before this - we want to limit to 5000 models, but this check should not happen here. In this location we can either relax the limit, or not do the check at all (no point in checking the length after the zip, since if after zip we have 1MB of data then we're way over the size limit, and the DB won't be able to store such an object).

Reply:
As discussed I change the limit to 10MB (only for sanity) & will implement the other check in https://iguazio.atlassian.net/browse/ML-8992